apps/public/pages/housing_counselors.tsx

```diff
@@ -0,0 +1,49 @@
+import Layout from "../layouts/application"
+import PageHeader from "@bloom/ui-components/src/headers/page_header/page_header"
+import HousingCounselor from "@bloom/ui-components/src/page_components/housing_counselor"
+import { HousingCounselor as Counselor } from "@bloom/ui-components/src/types"
+import { Component } from "react"
+import axios from "axios"
+
+interface HousingCounselorsProps {
+  counselors: Counselor[]
+}
+
+export default class extends Component<HousingCounselorsProps> {
+  public static async getInitialProps() {
```

Comment:
@bk3c this getInitialProps isn't running on this page for me. Any idea why?

Reply:
resolved it, I didn't realize the console errors were logging to my terminal instead of my browser.